common/msgs/autoware_control_msgs/msg/EmergencyMode.msg

```diff
@@ -0,0 +1 @@
+bool is
```

Comment:
The rationale for using a custom message was because the `Bool` message is [deprecated in Foxy](https://github.com/ros2/common_interfaces/blob/master/std_msgs/msg/Bool.msg), correct?

Reply:
To be honest I was under the impression that most primitive messages were removed in ROS2 altogether. (When I tried to use them at some point, it looked like they were and the docs seemed to confirm that, but I must have looked in the wrong places.) Still, as they're deprecated, it doesn't hurt to move away.